README.md

```diff
@@ -268,6 +268,26 @@ editors:
   - css_select: h1 > a
 ```
 
+#### `debug_files`
```

Comment:
Any thoughts about the name?

`debug_file` seems sometimes better but hides the fact that it can create multiple files in places like `rss`.

Reply:
I think the name is fine.